torchvision/prototype/datasets/_builtin/stanford_cars.py

```diff
@@ -0,0 +1,85 @@
+from typing import Any, Dict, List, Tuple, Iterator, BinaryIO
+
+from torchdata.datapipes.iter import Filter, IterDataPipe, Mapper, Zipper
+from torchvision.prototype.datasets.utils import Dataset, DatasetConfig, DatasetInfo, HttpResource, OnlineResource
+from torchvision.prototype.datasets.utils._internal import hint_sharding, hint_shuffling, path_comparator, read_mat
+from torchvision.prototype.features import BoundingBox, EncodedImage, Label
+
+
+class StanfordCarsLabelReader(IterDataPipe[Tuple[int, int, int, int, int, str]]):
+    def __init__(self, datapipe: IterDataPipe[Dict[str, Any]]) -> None:
+        self.datapipe = datapipe
+
+    def __iter__(self) -> Iterator[Tuple[int, int, int, int, int, str]]:
+        for _, file in self.datapipe:
+            data = read_mat(file, squeeze_me=True)
+            for ann in data["annotations"]:
+                yield tuple(ann)
```

Comment:
We can just ignore `mypy` here

```suggestion
                yield tuple(ann)  # type: ignore[misc]
```